package.json

```diff
@@ -11,7 +11,8 @@
     "test": "test"
   },
   "scripts": {
-    "start": "gulp",
+    "start": "(npm run createjson) & gulp",
+    "createjson": "cd ./src/main/create-operator-json/core && node --experimental-modules --es-module-specifier-resolution=node operatorModule.js",
```

Comment:
Would it be possible to make this a "gulp" command instead? I think it would be better to add this as a task for the two existing gulp commands if possible.

Reply:
That was my first thought, 
I read through that some articles and docs that gulp is a task runner and not meant to run scripts. The js I created is what creates a json file, or rather overwrites a json file inside a folder before the gulp task runs. I will look into it more, I did find that a third party gulp module can run a js script

Reply:
found the module: 'gulp-run'
Lets you run npm scripts.
I'll go forward with it